File: app/src/main/java/com/edotassi/amazmod/ui/SplashActivity.java
package com.edotassi.amazmod.ui;

import android.content.Intent;
import android.os.Bundle;

public class SplashActivity extends BaseAppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        Intent intent = new Intent(this, MainActivity.class);
        startActivity(intent);
        finish();
    }
}

File: app/src/main/java/com/edotassi/amazmod/ui/SplashActivity.kt
package com.edotassi.amazmod.ui

import android.content.Intent
import android.os.Bundle

class SplashActivity : BaseAppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val intent = Intent(this, MainActivity::class.java)
        startActivity(intent)
        finish()
    }
}